components/mediators/cache/org.wso2.carbon.mediator.cache/src/main/java/org/wso2/carbon/mediator/cache/CachingConstants.java

```diff
@@ -92,6 +93,12 @@
      */
     public static final boolean DEFAULT_ADD_AGE_HEADER = false;
 
+    /**
+     * Headers which needs to be permanently excluded in hashing due to its dynamic nature
```

Comment:
Doc commentst need to end with a period. Please refer to https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141999.html for coding conventions